src/main/java/mate/dao/ManufacturerDaoImpl.java

```diff
@@ -87,7 +87,7 @@ public Manufacturer update(Manufacturer manufacturer) {
 
     @Override
     public boolean delete(Long id) {
-        String query = "UPDATE manufacturers SET deleted = TRUE WHERE id = ?";
+        String query = "UPDATE manufacturers SET is_deleted = TRUE WHERE id = ?";
```

Comment:
next time it will be simplest to create a table from init_db.sql if all logic was wrote

Reply:
i'm fighting with deleted column from jdbc task)
I'll win